openstack/domain-seeds/values.yaml

```
@@ -35,3 +35,6 @@ tempestAdminPassword: ""
 # cc3test domain - user credentials
 cc3testAdminUser: DEFINE_IN_REGION_VALUES
 cc3testMemberUser: DEFINE_IN_REGION_VALUES
+
+domain:
+  deploy: true
```

Please use more descriptive field names, so that we can place the region-specific values into `globals.yaml` and also consume them from other service's charts:
```suggestion
domain_seeds:
  skip_hcm_domain: true
```
```